src/app/services/userService.tsx

```diff
@@ -0,0 +1,27 @@
+export const fetchUserProfile = async (userId: string) => {
```

Comment:
Yes, so I would just add a return type in the function that way you don't have to define that interface in the User page.
I used `Partial` just so Typescript knows not to expect all of the properties on a user object that comes through.

```suggestion
export const fetchUserProfile = async (userId: string): Promise<Partial<User>> => {
```